src/test/resources/goal-output/plume-lib-1.1.0/plume.StrTok_goal.json

```diff
@@ -0,0 +1,428 @@
+    },
+    "throwsTags": []
+  },
+  {
+    "signature": "commentChar(int ch)",
+    "name": "commentChar",
+    "containingClass": {
+      "qualifiedName": "plume.StrTok",
+      "name": "StrTok",
+      "isArray": false
+    },
+    "targetClass": "plume.StrTok",
+    "isVarArgs": false,
+    "returnType": {
+      "qualifiedName": "void",
+      "name": "void",
+      "isArray": false
+    },
+    "parameters": [
+      {
+        "type": {
+          "qualifiedName": "int",
+          "name": "int",
+          "isArray": false
+        },
+        "name": "ch"
+      }
+    ],
+    "paramTags": [
+      {
+        "parameter": {
+          "type": {
+            "qualifiedName": "int",
+            "name": "int",
+            "isArray": false
+          },
+          "name": "ch"
+        },
+        "comment": "the comment character",
+        "kind": "PARAM",
+        "condition": ""
+      }
+    ],
+    "throwsTags": []
+  },
+  {
+    "signature": "quoteChar(int ch)",
+    "name": "quoteChar",
+    "containingClass": {
+      "qualifiedName": "plume.StrTok",
+      "name": "StrTok",
+      "isArray": false
+    },
+    "targetClass": "plume.StrTok",
+    "isVarArgs": false,
+    "returnType": {
+      "qualifiedName": "void",
+      "name": "void",
+      "isArray": false
+    },
+    "parameters": [
+      {
+        "type": {
+          "qualifiedName": "int",
+          "name": "int",
+          "isArray": false
+        },
+        "name": "ch"
+      }
+    ],
+    "paramTags": [
+      {
+        "parameter": {
+          "type": {
+            "qualifiedName": "int",
+            "name": "int",
+            "isArray": false
+          },
+          "name": "ch"
+        },
+        "comment": "the quoting character",
+        "kind": "PARAM",
+        "condition": ""
+      }
+    ],
+    "throwsTags": []
+  },
+  {
+    "signature": "ttype()",
+    "name": "ttype",
+    "containingClass": {
+      "qualifiedName": "plume.StrTok",
+      "name": "StrTok",
+      "isArray": false
+    },
+    "targetClass": "plume.StrTok",
+    "isVarArgs": false,
+    "returnType": {
+      "qualifiedName": "int",
+      "name": "int",
+      "isArray": false
+    },
+    "parameters": [],
+    "paramTags": [],
+    "returnTag": {
+      "comment": "the type of the current token",
+      "kind": "RETURN",
+      "condition": ""
+    },
+    "throwsTags": []
+  },
+  {
+    "signature": "isWord()",
+    "name": "isWord",
+    "containingClass": {
+      "qualifiedName": "plume.StrTok",
+      "name": "StrTok",
+      "isArray": false
+    },
+    "targetClass": "plume.StrTok",
+    "isVarArgs": false,
+    "returnType": {
+      "qualifiedName": "boolean",
+      "name": "boolean",
+      "isArray": false
+    },
+    "parameters": [],
+    "paramTags": [],
+    "returnTag": {
+      "comment": "true iff the current token is a word (identifier)",
+      "kind": "RETURN",
+      "condition": "target.isWord() ? result == true"
```

Comment:
We decided to do not include the method under analysis in the specifications of the method itself.
In other words, specifications of the method `isWord` cannot include calls to `isWord` method.